model/patch_lifecycle.go

```diff
@@ -491,10 +492,27 @@ func FinalizePatch(ctx context.Context, p *patch.Patch, requester string, github
 	}
 	intermediateProject.CreateTime = patchVersion.CreateTime
 
-	manifest, err := constructManifest(patchVersion, project, projectRef, settings)
+	autoUpdateModules := project.GetAutoUpdateModules()
+
+	mfst, err := constructManifest(patchVersion, projectRef, autoUpdateModules, settings)
 	if err != nil {
 		return nil, errors.Wrap(err, "constructing manifest")
 	}
+	if mfst != nil {
+		baseManifest, err := manifest.FindFromVersion(patchVersion.Id, patchVersion.Identifier, patchVersion.Revision, patchVersion.Requester)
```

Comment:
Can this logic be moved into constructManifest?

Reply:
oh I see that constructManifest is also used by CreateManifest but I think it could still be a paramter for constructManifest or it's own helper function.